lib/pejs.ts

```diff
@@ -102,14 +103,37 @@ export class PEJS {
    * @param presentationDefinition the definition of what is expected in the presentation.
    * @param selectedCredential the credentials which were declared selectable by getSelectableCredentials and then chosen by the intelligent-user
    * (e.g. human).
+   * @param holderDID the decentralized identity of the wallet holder. This is used to identify the credentials issued to the holder of wallet.
    *
    * @return the presentation submission.
    */
-  public submissionFrom(
+  public presentationFrom(
     presentationDefinition: PresentationDefinition,
-    selectedCredential: VerifiableCredential[]
-  ): PresentationSubmission {
-    return this._evaluationClientWrapper.submissionFrom(presentationDefinition, selectedCredential);
+    selectedCredential: VerifiableCredential[],
+    holderDID: string
+  ): Presentation {
+    const presentationSubmission = this._evaluationClientWrapper.submissionFrom(
+      presentationDefinition,
+      selectedCredential
+    );
+
+    return PEJS.getPresentation(holderDID, presentationSubmission, selectedCredential);
+  }
+
+  private static getPresentation(
+      holderDID: string,
```

Comment:
Should this be static?

Reply:
It can be static. It does not have to non-static. And it is working.